.github/workflows/format-and-build.yml

```diff
@@ -22,13 +22,21 @@ jobs:
         with:
           node-version: 20.x
 
+      - name: 🔧 Configure Git
+        run: |
+          git config user.name 'github-actions[bot]'
+          git config user.email 'github-actions[bot]@users.noreply.github.com'
+
       - name: 📦 Install Dependencies
         run: npm install
 
-      - name: 🧹 Format
+      - name: 🔄 Format and build
         run: |
           npm run format
           npm run build
 
-      - name: 💾 Commit
-        uses: autofix-ci/action@551dded8c6cc8a1054039c8bc0b8b48c51dfc6ef
+      - name: 💾 Commit changes
+        run: |
+          git add .
+          git commit -m "chore: format and build"
+          git push
```

Comment:
one reason why people use autofix is that it works on external prs because the default permissions for pull_request get nerfed on external, so this will just always throw an error for those prs instead

Reply:
> @okineadev

Ok?...

Reply:
> one reason why people use autofix is that it works on external prs because the default permissions for pull_request get nerfed on external, so this will just always throw an error for those prs instead

In [`a00108b` (#121)](https://github.com/oven-sh/setup-bun/pull/121/commits/a00108bc0a9c6039f44c6655af481e8621c0d906) the code now builds and commits only before release, so this is now irrelevant